doc/man7/Ed25519.pod

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
This new line should not be inserted, and this also broke 'doc-nit' check in travis-ci.